torch/onnx/_internal/fx/export.py

```diff
@@ -719,21 +745,17 @@ def _replace_get_attr_with_placeholder(graph_module: torch.fx.GraphModule):
 
             replaced_attrs.append(replaced_attr)
 
-    return replaced_attrs
+    return tuple(replaced_attrs)
 
 
-def export_without_parameters_and_buffers(
+@_beartype.beartype
+def _trace_into_fx_graph_via_fx_symbolic_trace(
     module: torch.nn.Module,
     *args,
-    decomposition_table: Dict[torch._ops.OpOverload, Callable] = None,
```

Comment:
Moved tracing code from `export_without_parameters_and_buffers` to `_trace_into_fx_graph_via_fx_symbolic_trace`.